src/server/hook_func.c

```diff
@@ -4697,20 +4730,24 @@ recreate_request(struct batch_request *preq)
 	hook_output_param_init(&req_params);
 	if (preq->rq_type == PBS_BATCH_QueueJob) {
 		req_params.rq_job = (struct rq_quejob *)&preq->rq_ind.rq_queuejob;
+		snprintf(perf_label, sizeof(perf_label), "hook_%s_%s_%d", HOOKSTR_QUEUEJOB, preq->rq_ind.rq_queuejob.rq_jid, getpid());
 		rc = pbs_python_event_to_request(HOOK_EVENT_QUEUEJOB,
-			&req_params);
+			&req_params, perf_label, "hook_output");
```

Comment:
Should the "hook_output" string be a macro?

Reply:
Yes...updated.